src/Domain/BlockLogicalCoordinates.cpp

```diff
@@ -60,14 +60,33 @@ std::vector<block_logical_coord_holder<Dim>> block_logical_coordinates(
           } else {
             continue;  // Not in this block
           }
-        } else {  // frame is different than ::Frame::Inertial
+        } else if constexpr (std::is_same_v<Frame, ::Frame::Distorted>) {
+          // Point is in the distorted frame, so we need to map to the grid
+          // frame and then the logical frame.
+          const auto moving_inv =
+              block.moving_mesh_grid_to_distorted_map().inverse(
```

Comment:
Please add an assert that `block.has_distorted_frame()`. Also, I think the inertial coord case in the branch above needs to handle the distorted frame, right? Please test this case where you have a distorted frame and want to invert an inertial coord.

Reply:
The inertial coord case above is fine as is; it doesn't matter if there is a distorted frame or not.

However, the new distorted frame case does need to be changed, because the current case will fail if some blocks don't have distorted frames.

Reply:
Update from the call today: Would be good to add some docs to Block about which maps are available.

Reply:
I found a better way to treat this case.  See the long comment in BlockLogicalCoordinates.cpp